fontforge/python.c

```diff
@@ -5268,11 +5426,11 @@ static SplineSet *_SSFromLayer(PyFF_Layer *layer, int flags) {
     return( head );
 }
 
-static SplineSet *SSFromLayer(PyFF_Layer *layer) {
+SplineSet *SSFromLayer(PyFF_Layer *layer) {
 	return _SSFromLayer(layer, pconvert_flag_none);
 }
 
-static PyFF_Layer *LayerFromSS(SplineSet *ss,PyFF_Layer *layer) {
+PyFF_Layer *LayerFromSS(SplineSet *ss,PyFF_Layer *layer) {
```

Comment:
Why are these now non-static?

Reply:
Hmm. I'm guessing I used these in `pythonui.c` and then that went away in a later change. I think I can reverse it.